build.gradle.kts

```diff
@@ -59,17 +59,8 @@ val google_protobuf = "com.google.protobuf:protobuf-java:3.16.3"
 val google_guava = "com.google.guava:guava:31.1-jre"
 val google_tink = "com.google.crypto.tink:tink:1.5.0"
 
-val aws_core = "com.amazonaws:aws-java-sdk-core:1.12.472"
-val aws_s3 = "com.amazonaws:aws-java-sdk-s3:1.12.470"
-val aws_sts = "com.amazonaws:aws-java-sdk-sts:1.12.471"
-val aws_dynamodb = "com.amazonaws:aws-java-sdk-dynamodb:1.12.472"
-val aws_sns = "com.amazonaws:aws-java-sdk-sns:1.12.472"
-val aws_kms = "com.amazonaws:aws-java-sdk-kms:1.12.429"
-val aws_cloudwatch = "com.amazonaws:aws-java-sdk-cloudwatch:1.12.472"
-val aws_lambda = "com.amazonaws:aws-java-sdk-lambda:1.12.472"
-val aws_lambda_core = "com.amazonaws:aws-lambda-java-core:1.2.2"
-val aws_lambda_log4j = "com.amazonaws:aws-lambda-java-log4j2:1.5.1"
-val amazon_sns = "software.amazon.awssdk:sns:2.20.69"
+val aws_bom = "software.amazon.awssdk:bom:2.21.1"
```

Comment:
why 2.21.1 which is 6+ months older version. I suggest using latest version 2.25.16 (unless any problem with it)

Reply:
fixed, upgraded to 2.25.19